torch/csrc/jit/codegen/cuda/shape_inference.cpp

```diff
@@ -15,16 +15,117 @@ namespace cuda {
 
 namespace {
 
+std::vector<int64_t> getTensorSizes(TensorTypePtr const& tensor_type) {
+  TORCH_INTERNAL_ASSERT(tensor_type != nullptr, "Input must be a Tensor.");
+
+  TORCH_INTERNAL_ASSERT(
+      tensor_type->isComplete(), "Incomplete TensorType information.");
```

Comment:
`isComplete()` seems to be an overkill. We don't care much about stride, do we?

Reply:
e.g. if we only wanted tensor sizes, then we should only check for tensor sizes.